velox/exec/tests/TableScanTest.cpp

```diff
@@ -622,6 +622,36 @@ TEST_F(TableScanTest, count) {
   EXPECT_EQ(numRead, 10'000);
 }
 
+TEST_F(TableScanTest, batchSize) {
+  // One row is larger than 1KB. Total input is less than 10MB.
+  auto numColumns = 2 * 1024 / sizeof(int64_t);
```

Comment:
This is a bit difficult to understand. I wonder if there is a way to write this more clearly.

```
// Make a wide row of many BIGINT columns to ensure that row size is larger than 1KB.
auto columnSize = sizeof(int64_t);
auto rowSize = 2024; // 2K
auto numColumns = 1.5 * rowSize / columnSize;
```

Reply:
Why "1.5 *"?